index.html

```diff
@@ -0,0 +1,212 @@
+            <a
+              href=""
+              class="social-media"
+              target="_blank"
+              ><i class="fa-brands fa-facebook-f"></i>
+            </a>
+          </li>
+          <li>
+            <a
+              href=""
+              class="social-media"
+              target="_blank"
+              ><i class="fa-brands fa-twitter"></i
+            ></a>
+          </li>
+          <li><a href="#" class="social-media">English</a></li>
+          <li><a href="index.html" class="social-media">Mypage</a></li>
+          <li><a href="#" class="social-media">Logout</a></li>
+        </ul>
+      </header>
+      <nav class="nav">
+        <a class="logo" href="#home"
+          ><img
+            src="./images/phising.png"
+            alt="logo for phishing"
+        /></a>
+        <div class="hamburger" id="home">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <section class="navi-menu">
+          <a class="navi-link home" href="#home">Home</a>
+          <a class="navi-link" href="about.html">About</a>
+          <a class="navi-link" href="#things">Top 5 Social Engineering Skills</a>
+          <a class="navi-link" href="#program">Social engineers</a>
+          <a class="navi-link sponsor" href="#sponsor">Tutorials</a>
+        </section>
+      </nav>
+      <nav class="nav-desk">
+        <a class="logo-desk" href="#home"
+          ><img
+            src="./images/phising.png"
+            alt="logo for phishing"
+        /></a>
+        <section class="navi-menu-desk">
+          <a class="navi-link-desk home" href="#home">Home</a>
+          <a class="navi-link-desk" href="about.html">About</a>
+          <a class="navi-link-desk" href="#things">Top 5 Social Engineering Skills</a>
+          <a class="navi-link-desk" href="#join-desk">Social engineers</a>
+          <a class="navi-link-desk sponsor" href="#sponsor">Tutorials</a>
+        </section>
+      </nav>
+
+      <div class="main-desk">
+        <div class="intro">
+          <h4 class="hello">"Hello! World"</h4>
+          <h1 class="main-header">
+            SOCIAL ENGINEERING
+          </h1>
+        </div>
+        <p class="info">
+           Kevin Mitnick: There is no patch for stupidity.” is best saying to explain the impact of social engineering on security.
+           Though firewalls act to fire the walls we can not fire individuals inside our company with computer instructions. The only thing we
+           will do is to train our employees not to fall for socially engineered hackers.
+        </p>
+        <div class="time">
+          <h2 class="interval">2022.10.17(MON) ~ 30(SUN)</h2>
+          <p class="place">@Socialengineering</p>
+        </div>
+      </div>
+    </section>
+    
+    <section class="main1">
+      <div class="main-program" id="program">
+          <h2>social engineering skills</h2>
+          <div class="line"></div>
```

Comment:
You have done great at changing the background as requested by the previous reviewer. There are a few issues

- you seem to have lost the centralization of the underline of the header. Kindly centralize it.
- please let each icon for the different cards be different
- consider adjusting the margin above and below the title in each card as shown in the illustration
- give some opacity to the background of each card so that the background
- kindly use [this image](https://user-images.githubusercontent.com/20554583/211139733-cdb7f3bb-761d-4de8-ad46-3b830d76de27.jpg) for the background of the section 

Your design:
![image](https://user-images.githubusercontent.com/20554583/211139464-4c7d4adf-9213-4473-8cd2-3fb65e575529.png)


Figma design:
![image](https://user-images.githubusercontent.com/20567503/210991898-6d18ea9a-b1a2-45a7-a6a9-87c79145642d.png)